test/teslamate_web/controllers/drive_controller_test.exs

```diff
@@ -0,0 +1,99 @@
+defmodule TeslaMateWeb.DriveControllerTest do
+  use TeslaMateWeb.ConnCase
+
+  alias TeslaMate.Log
+  alias TeslaMate.Repo
+
+  def car_fixture(attrs \\ %{}) do
+    {:ok, car} =
+      attrs
+      |> Enum.into(%{efficiency: 0.153, eid: 42, model: "M3", vid: 42, vin: "xxxxx"})
+      |> Log.create_car()
+
+    car
+  end
+
+  defp drive_fixture(car) do
+    {:ok, drive} = Log.start_drive(car)
+
+    Log.insert_position(drive, %{
+      date: DateTime.utc_now(),
+      latitude: 5.0,
+      longitude: 5.0,
+      elevation: 100
+    })
+
+    Log.insert_position(drive, %{
+      date: DateTime.utc_now(),
+      latitude: 10.0,
+      longitude: 10.0,
+      elevation: 200
+    })
+
```

Comment:
To test the date-based sorting, a third `Position` with a date in the past could be inserted here.

Reply:
Done as suggested :)